edgelet/iotedge/src/check/checks/aziot_edged_version.rs

```diff
@@ -65,7 +61,7 @@ impl Checker for AziotEdgedVersion {
                 Err(err) => return CheckResult::Failed(err.into()),
             };
 
-            let request = hyper::Request::get("https://aka.ms/latest-iotedge-stable")
+            let request = hyper::Request::get("https://aka.ms/latest-aziot-stable-non-lts")
```

Comment:
`https://aka.ms/latest-iotedge-stable-non-lts`

Reply:
I put the latest aziot-edge in latest-versions-non-lts.json, so that's the correct file.

I think eventually we'll have latest-versions-non-lts.json and latest-versions-lts.json, with all packages listed in those files. Though that's probably more of a PM question.

Reply:
Nit: can we use latest-aziot-stable and latest-aziot-lts? (instead of non-lts, which seems inverted to me).

Reply:
How about latest-versions.json and latest-versions-lts.json? 1.1 is using latest-versions.json so I don't want to do anything that affects that.

Reply:
>I put the latest aziot-edge in latest-versions-non-lts.json, so that's the correct file.

We want to be able to release them independently, so no, it's not the correct file and not the correct URL.

Reply:
>Nit: can we use latest-aziot-stable and latest-aziot-lts? (instead of non-lts, which seems inverted to me).

You can do that for the JSON files in the azure-iotedge repo, but not the URLs. And I think keeping the URLs and the corresponding filenames in sync is cleaner.

Reply:
So the files should be:

https://aka.ms/latest-iotedge-stable --> latest_versions.json
```
{
    "iotedged": "1.1.1",
    "azureiotedge-agent": "1.1.1",
    "azureiotedge-hub": "1.1.1",
    "aziot-edge": "1.2.0"
}
```

https://aka.ms/latest-aziot-stable-non-lts --> latest-versions-non-lts.json
```
{
    "aziot-identity-service": "1.2.0"
}
```